Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py

```diff
@@ -707,8 +714,13 @@ def wildfire_get_sample_command():
     inputs = hash_args_handler(sha256, md5)
 
     for element in inputs:
-        result = wildfire_get_sample(element)
-
+        result, not_found = wildfire_get_sample(element)
```

Comment:
seems you do not use `result` when `not_found` is true.
so you dont need to catch `NotFoundError` in `wildfire_get_sample` and no need to pass not_found, but just catch it here and return the message hen you catch it.